server/confdata-stats.h

```diff
@@ -17,6 +17,7 @@ struct ConfdataStats : private vk::not_copyable {
 
   std::chrono::nanoseconds initial_loading_time{std::chrono::nanoseconds::zero()};
   std::chrono::nanoseconds total_updating_time{std::chrono::nanoseconds::zero()};
+  std::chrono::nanoseconds time_since_last_update{std::chrono::nanoseconds::zero()};
```

Comment:
Is it really time since last update? Looks like it time between two previous updates